### PR TITLE
Munch dev searchbar

### PR DESCRIPTION
**Pull request by xulean**
add /searchbar

### CODE REVIEW

File: backend/src/index.ts [hunk 1]
```diff
@@ -33,7 +35,7 @@ AppDataSource.initialize().then(async () => {
 const app = express()
 
 const corsOptions = {
-    origin: 'http://127.0.0.1:3000',
```
Comment (austinhedrick548) [inline backend/src/index.ts:36]: Can you change this back to 127.0.0.1:3000? Localhost is how to get it working locally, but we have it set to the actual IP normally (the reason escapes me)

File: backend/yarn.lock
Comment (austinhedrick548) [inline backend/yarn.lock:1]: Don't think we want to send the yarn.lock files up

File: backend/src/index.ts (hunk 1 above)
Comment (austinhedrick548) [inline backend/src/index.ts]: Can you change the IP back to the 127... IP?

File: backend/yarn.lock
Comment (xulean) [reply backend/yarn.lock:1]: deleted

**Review by austinhedrick548 — APPROVED** [review]
Code looks good to me

**Comment by austinhedrick548** [comment]
Closed since duplicate code merged elsewhere

**Pull request closed without merging**